sdks/python/apache_beam/testing/util.py

```diff
@@ -141,6 +145,21 @@ def _equal(actual):
   return _equal
 
 
+def matches_all(expected):
```

Comment:
Is this different from `contains_in_any_order` above?

Reply:
The `contains_in_any_order` seems to require a set of hashable items. It simply does `collections.Counter()` on the iterable, and compares both counters. `matches_all`  takes in an iterable of elements or matchers, and matches them 1:1 properly.

Reply:
gotcha